.github/workflows/CI-integrationtests.yml

```diff
@@ -42,4 +42,4 @@ jobs:
           source /workdir/env/bin/activate
           pip install '.[tests,dev,doc]'
           export PYTHONPATH=$PYTHONPATH:.
-          pytest --color=yes --durations=10 --no-cov tests/integration_tests/
+          pytest --color=yes --durations=10 -n auto --dist loadscope --no-cov tests/integration_tests/
```

Comment:
Is there a reason why you group the tests by module and not in random order using load?

Reply:
No strong reasons - it guarantees that all tests in a module run in the same process. I have used the ctapipe CI as a guidance here. Do you think we should change this?

Reply:
I don't have a strong optionion on this at the moment. We can let it like this and add the new CI we just discussed.

Reply:
see also issue #834